aws_lambda_powertools/utilities/parameters/base.py

```diff
@@ -159,6 +163,45 @@ def _get_multiple(self, path: str, **sdk_options) -> Dict[str, str]:
         raise NotImplementedError()
 
 
+def get_transform_method(key: str, transform: str) -> Optional[str]:
```

Comment:
Should `transform` be an optional parameter here or a nullable value?
E.g. similar to how `transform` get passed to the get() and get_multiple() methods.

Reply:
@nmoutschen - is currently only called from `get_multiple` and when the `transform is not None`

```python3
        if transform is not None:
            for (key, value) in values.items():
                _transform = get_transform_method(key, transform)
```

I am not sure if it makes sense to use `auto` for single items.

Reply:
My idea here was that when I pass `transform=None`, I'd get `None` as a return value.

It might not be needed today, but I'm thinking from two point of view: consistency with other function signatures (excepted behaviours for contributors), and if we want to move this transform if it's useful for other utilities in the future.

Looking at the code today, it would only change the signature since we already return the transform if it's not "auto":

```python
    if transform != "auto":
        return transform
```

Reply:
@nmoutschen I made this change. You can check again.